README.md

```diff
@@ -3,31 +3,30 @@
 [![CircleCI](https://circleci.com/gh/HDE/arsenic/tree/master.svg?style=svg)](https://circleci.com/gh/HDE/arsenic/tree/master) [![Documentation Status](https://readthedocs.org/projects/arsenic/badge/?version=latest)](http://arsenic.readthedocs.io/en/latest/?badge=latest)
 
 
-Asynchronous, framework-independent webdriver client for AsyncIO/Aiohttp and
-Tornado.
+Asynchronous webdriver client built on asyncio.
 
 
 ## Quickstart
 
-Let's run Firefox via Geckodriver using Aiohttp.
+Let's run a local Firefox instance.
 
 
 ```python
 
-from arsenic.engines.aiohttp import Aiohttp
+from arsenic import get_session
 from arsenic.browsers import Firefox
 from arsenic.services import Geckodriver
 
 
 async def example():
-    # start geckodriver using aiohttp/asyncio
-    async with Geckodriver().run(Aiohttp) as driver:
-        # start a new browser session
-        async with driver.session(Firefox()) as session:
-            # go to example.com
-            await session.get('http://example.com')
-            # wait up to 5 seconds to get the h1 element from the page
-            h1 = await driver.wait_for_element(5, 'h1')
-            # print the text of the h1 element
-            print(await h1.get_text())
+    # Runs geckodriver and starts a firefox session
+    async with get_session(Geckodriver(), Firefox()) as session:
```

Comment:
`Firefox` is a callable on purpose. You can pass it extra stuff (100% undocumented now) to instruct the webdriver server to start the browser in a specific way. For example, you can select a specific Profile in Firefox or override the homepage etc. 

There is absolutely an argument to be had for a higher-level interface though which handles service+browser. `get_browser('firefox')` or `get_driver('remote', 'firefox')` may not be a bad API, though I personally really dislike magical strings.